### PR TITLE
deps(quic): update quinn to 0.11.6

### PR DESCRIPTION
**Pull request by kalabukdima**
## Description

Among other changes, it includes a fix for this issue which often reproduces with libp2p: https://github.com/quinn-rs/quinn/issues/1889

## Notes & open questions

The list of changes: https://github.com/quinn-rs/quinn/compare/quinn-0.11.2...quinn-0.11.6

## Change checklist

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] A changelog entry has been made in the appropriate crates

### CODE REVIEW

**Review by dariusc93 — COMMENTED** [review]
Thanks for the PR. I wonder if we should bump it to `0.11.6` since there seems to be additional changes and possible fixes up that point as well. Thoughts?

**Comment by kalabukdima** [comment]
> Thanks for the PR. I wonder if we should bump it to `0.11.6` since there seems to be additional changes and possible fixes up that point as well. Thoughts?

Sure, I would gladly update to the latest version! I was just not sure what is libp2p's policy for dependency updates. I didn't review the changes, but hopefully "fix" version updates won't break anything.

File: transports/quic/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "libp2p-quic"
-version = "0.11.2"
+version = "0.11.3"
```
Comment (elenaf9) [inline transports/quic/Cargo.toml]:
```suggestion
version = "0.11.2"
```

`v0.11.2` isn't released yet

File: Cargo.toml [hunk 2]
```diff
@@ -93,7 +93,7 @@ libp2p-perf = { version = "0.4.0", path = "protocols/perf" }
 libp2p-ping = { version = "0.45.1", path = "protocols/ping" }
 libp2p-plaintext = { version = "0.42.0", path = "transports/plaintext" }
 libp2p-pnet = { version = "0.25.0", path = "transports/pnet" }
-libp2p-quic = { version = "0.11.2", path = "transports/quic" }
+libp2p-quic = { version = "0.11.3", path = "transports/quic" }
```
Comment (elenaf9) [inline Cargo.toml]:
```suggestion
libp2p-quic = { version = "0.11.2", path = "transports/quic" }
```

File: transports/quic/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+## 0.11.3
+
+- Update `quinn` to 0.11.6
+  See [PR 5757](https://github.com/libp2p/rust-libp2p/pull/5757).
+
```
Comment (elenaf9) [inline transports/quic/CHANGELOG.md]:
```suggestion
```

No need for a changelog entry.

**Review by elenaf9 — COMMENTED** [review]
Thanks!

Version bump isn't needed because we didn't release `libp2p-quic-0.11.2` yet.

**Review by elenaf9 — APPROVED** [review]
Thanks!

**Pull request merged**